beamerthemeuulm.sty

```diff
@@ -123,9 +123,9 @@
 \universitylogo{uulm}
 \clearinstitutelogo
 
-\newcommand{\createtitleslide}[2]{
+\newcommand{\createtitleslide}{
     {
-    \ifx \uulmtitlepicture \empty \else \usebackgroundtemplate{\includegraphics[trim=0 0 0 #2,clip,width=\paperwidth]{#1}} \fi
+    \ifx \uulmtitlepicture \empty \else \usebackgroundtemplate{\includegraphics[trim=0 0 0 \uulmtitlepictureoffset,clip,width=\paperwidth]{\uulmtitlepicture}} \fi
```

Comment:
I would not make createtitleslide (-> `\@createtitleslide`) user accessible. Furthermore, againtitle (they do the same now :D), may check if there was some previously issued maketitle command? See below